pages/AlltimeFavorites/index.js

```diff
@@ -0,0 +1,126 @@
+import PreviewPicture from "../../components/PreviewPicture";
+import { Container } from "../../components/styledComponents/Container";
+import { StyledButton } from "@/components/styledComponents/StyledButton";
+import React from "react";
+import useSWR from "swr";
+import Loading from "../../components/Loading";
+import { useAtom } from "jotai";
+import { isFavorit } from "../../store/isFavorit";
+
+const fetcher = (url) => fetch(url).then((res) => res.json());
+
+async function increaseFavtoMongoDB(picSRCCloudinary, picID) {
+  await fetch(`/api/Favorites/alltimeFavorites/increaseFavorites/${picID}`, {
+    method: "PUT",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({ picSRCCloudinary }),
+  });
+
+  return;
+}
+
+async function decreaseFavtoMongoDB(picSRCCloudinary, picID) {
+  await fetch(`/api/Favorites/alltimeFavorites/decreaseFavorites/${picID}`, {
+    method: "PUT",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({ picSRCCloudinary }),
+  });
+
+  return;
+}
```

Comment:
These functions could be combined into one in order to avoid code duplication:

```js
async function updateFavorites(picSRCCloudinary, picID, action) {
  const url = `/api/Favorites/alltimeFavorites/${action}/${picID}`;
  const response = await fetch(url, {
    method: "PUT",
    headers: {
      "Content-Type": "application/json",
    },
    body: JSON.stringify({ picSRCCloudinary }),
  });
  return response;
}
```